ggshield/verticals/secret/output/messages.py

```diff
@@ -1,6 +1,9 @@
 from ggshield.core.text_utils import STYLE, format_text
+from textwrap import TextWrapper
 
-
+def pad_text(text: str, padding: str ='  '):
+   return TextWrapper(initial_indent=padding,subsequent_indent=padding).indent(text)
```

Comment:
`TextWrapper` does not have an `indent()` method. You need to change your code to:
1. `import textwrap` instead of `from textwrap import TextWrapper`
2. replace this line with `textwrap.indent(text, "  ")`